cypress/lib/helper.js

```diff
@@ -51,13 +51,19 @@ export function checkBookingsOrder(expectedData) {
 export function createBookingsDataTest() {
   const bookingsDataTest = []
   for (let i = 1; i <= 20; ++i) {
+    let teamMember = 'admin user'
+    if (i % 2 == 0) {
+      teamMember = 'normal user'
+    }
+
     bookingsDataTest[i - 1] = [
```

Comment:
can we change this array to a key value array?

Reply:
if we do that, the "checkBookingsOrder" function would have to be modified, and that would break the other tests